stripe/src/main/java/com/stripe/android/view/ExpiryDateEditText.kt

```diff
@@ -210,6 +210,11 @@ class ExpiryDateEditText @JvmOverloads constructor(
                         isDateValid = false
                     }
 
+                    setErrorMessage(
+                        resources.getString(
+                            if (!expirationDate.isMonthValid) R.string.invalid_expiry_month else R.string.invalid_expiry_year
+                        )
+                    )
```

Comment:
Nice, can we now remove the `setErrorMessage()` call in `init {}`?

Reply:
Done